.github/workflows/hadolint-lint.yml

```
@@ -0,0 +1,10 @@
+name: Hadolint Dockerfile Lint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 
+      - uses: hadolint/hadolint-action@v2.0.0
```

```suggestion
      - uses: hadolint/hadolint-action@f988afea3da57ee48710a9795b6bb677cc901183 # v2.1.0
```

Other concerns I mentioned in https://github.com/openedx/edx-cookiecutters/pull/265/files#r984133670 don't seem to apply here; the version of hadolint itself is pinned by the Action.